datapacks/fleet/entities/battleship.json

```diff
@@ -1,12 +1,16 @@
 {
     "name": "Battleship",
-    "max_health": 200,
+    "health": {
+      "max": 200,
+      "regen": 10
+    },
     "movement": {
         "range": 1
       },
     "combat": {
-        "damage": "100",
-        "range": 4
+      "range": 4,
+      "attack": "100",
```

Comment:
This should be a number not a string. The parsing should have detected this - I will need to look into why it didn't

Reply:
Thanks, fixed in 3dbbd96